infrastructure/validators/rewards-payout.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Rewards Payout
+description: Learn how validator rewards work on the network, including era points, payout distribution, running multiple validators, and nominator payments.
+---
+# Rewards Payout
+
+## Introduction
+
+Understanding how rewards are distributed to validators and nominators is essential for network participants. In Polkadot and Kusama, validators earn rewards based on their era points, which are accrued through actions like block production and parachain validation. This guide explains the payout scheme, factors influencing rewards, and how multiple validators affect returns. Validators can also share rewards with nominators, who contribute by staking behind them. By following the payout mechanics, validators can optimize their earnings and better engage with their nominators.
+
+## Era Points
+
+The Polkadot ecosystem measures their reward cycles in a unit called an era. Kusama eras are approximately 6 hours long, and Polkadot eras are 24 hours. At the end of each era, validators are paid proportionally to the amount of [era points](TODO: path) they have collected. Era points are reward points earned for payable actions like:
+
+- Issuing validity statements for [parachain](TODO: path) blocks
+- Producing a non-uncle block in the relay chain
+- Producing a reference to a previously unreferenced uncle block
+- Producing a referenced uncle block
+
+!!!note
+    An uncle block is a relay chain block that is valid in every regard but has failed to become canonical. This can happen when two or more validators are block producers in a single slot, and the block produced by one validator reaches the next block producer before the others. The lagging blocks are called uncle blocks.
+
+Payments occur at the end of every era.
+
+## Reward Variance
+
+Rewards in Polkadot and Kusama staking systems can fluctuate due to differences in era points earned by para-validators and non-para-validators. Para-validators generally contribute more to the overall reward distribution due to their role in validating parachain blocks, thus influencing the variance in staking rewards.
+
+To illustrate this relationship:
+
+- Para-validator era points tend to have a higher impact on the expected value of staking rewards compared to non-para-validator points
+- The variance in staking rewards increases as the total number of validators grows relative to the number of para-validators
+- In simpler terms, when more validators are added to the active set without increasing the para-validator pool, the disparity in rewards between validators becomes more pronounced
+
+However, despite this increased variance, rewards tend to even out over time due to the continuous rotation of para-validators across eras. The network's design ensures that over multiple eras, each validator has an equal opportunity to participate in para-validation, eventually leading to a balanced distribution of rewards.
+
+??? function "Probability in Staking Rewards"
```

Comment:
I think either a `note` or `interface` admonition might look a little bit better in this case

Reply:
Looks gooood! ✨